tracks/bash/exercises/pangram/mentoring.md

```diff
@@ -0,0 +1,42 @@
+## Common Suggestions
+
+* use brace expansion
+
+```md
+On LN, consider a more concise way to provide a sequence of letters.
+[Brace Expansion](https://www.gnu.org/software/bash/manual/html_node/Brace-Expansion.html)
+in the Bash Reference Manual might be useful.
+```
+
+## General Guidelines
+
+* there is no `main` function
+
+```md
+Generally, you should encapsulate the main body of your script in a `main`
+function, like you may remember from the Hello World exercise.  It
+encapsulates a chunk of logic in one function which is good practice for when
+things get bigger.
+```
+
+* `[` is used
+
+```md
+In bash, prefer `[[`/`]]` over `[`/`]`. It's more powerful than `[`/`]` and less likely to act
+in unexpected ways.
+```
+
+* there is no shebang
+
+```md
+There is no  shebang on the top line. This means if you `chmod +x` your
```

Comment:
```suggestion
There is no shebang on the top line. This means if you `chmod +x` your
```

We might be doing single sentence per line as well, to avoid noisy diffs.